elogia_sale/models/elogia_sale.py

```diff
@@ -521,11 +563,9 @@ def action_view_sales(self):
             'type': 'ir.actions.act_window',
             'view_mode': 'form',
             'res_model': 'sale.order',
-            'domain': [('control_id', '=', self.id)],
+            'domain': [('id', 'in', self.order_ids.mapped('order_id').ids)],
```

Comment:
mejor así:
`'domain': [('id', 'in', self.order_ids.mapped('order_id').mapped('id'))],`
con el mapped si en el medio alguno está vacío no da error